### ISSUE TITLE
Accuracy remain unchanged for some class

### ISSUE THREAD

**Issue by benlam62** [BVLC/caffe#2622]
Hi folks, during the VGG-16 net training for 2 classes + background class, I find weird that one of the class' accuracy remain unchanged. You can see the output #1 below remain 0.333. Does anyone has any comment on it?

I0618 23:45:36.158685 40897 solver.cpp:209] Iteration 54, loss = 0.767131
I0618 23:45:36.158941 40897 solver.cpp:224]     Train net output #0: accuracy = 0.734801
I0618 23:45:36.158967 40897 solver.cpp:224]     Train net output #1: accuracy = 0.333333
I0618 23:45:36.158982 40897 solver.cpp:224]     Train net output #2: accuracy = 0.244934
I0618 23:45:36.159001 40897 solver.cpp:447] Iteration 54, lr = 0.001
I0618 23:56:05.718288 40897 solver.cpp:209] Iteration 55, loss = 0.740111
I0618 23:56:05.718528 40897 solver.cpp:224]     Train net output #0: accuracy = 0.74539
I0618 23:56:05.718552 40897 solver.cpp:224]     Train net output #1: accuracy = 0.333333
I0618 23:56:05.718569 40897 solver.cpp:224]     Train net output #2: accuracy = 0.248463
I0618 23:56:05.718587 40897 solver.cpp:447] Iteration 55, lr = 0.001
I0619 00:06:34.629878 40897 solver.cpp:209] Iteration 56, loss = 0.710542
I0619 00:06:34.630365 40897 solver.cpp:224]     Train net output #0: accuracy = 0.76395
I0619 00:06:34.630394 40897 solver.cpp:224]     Train net output #1: accuracy = 0.333333
I0619 00:06:34.630411 40897 solver.cpp:224]     Train net output #2: accuracy = 0.25465
I0619 00:06:34.630430 40897 solver.cpp:447] Iteration 56, lr = 0.001

Thx
Ben

**Comment by longjon** [comment]
This tracker is for issues and bugs with Caffe. Please ask usage and modeling questions on the [caffe-users](https://groups.google.com/forum/#!forum/caffe-users) list. Thanks!
